sdk/ai/azure-ai-projects/README.md

```diff
@@ -39,6 +39,232 @@ Use the returned token credential to authenticate the client:
 
 ## Examples
 
+### Agents
+
+The following steps outline the typical sequence for interacting with agents:
+
+1. Create a project client.
+2. Upload files and associate them with a vector store for file search.
+3. Create a thread.
+4. Create a message.
+5. Execute `Run`, `Run_and_Process`, or `Stream`.
+6. Clean up by deleting resources.
+
+Additionally, you can retrieve messages.
+
+#### Create Project Client
+
+To create a project client, use the following example:
+
+<!-- SNIPPET:sample_agents_basics.create_project_client -->
+
+```python
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+```
+
+<!-- END SNIPPET -->
+
+#### Create Vector Store and Upload File
```

Comment:
In the doc we could have following structure

Agents Operations:
- Create agent
- Create thread
- Create message
- Create run or stream
- List/Get messages

Agent Tools
- Functions
- FileSearch
- CodeInterpreter
- BingGrounding
- Other tools..

Vectorstore would be under FileSearch tool there.

Reply:
Could you take a look the new structure I just pushed.